src/Compiler/FileNameAndLineNumberAddingPreCompiler.php

```diff
@@ -52,9 +52,6 @@ public function compileString(string $value): string
         return implode(PHP_EOL, $lines);
     }
 
-    /**
-     * @todo remove fluent, make service method with fileName as argument to avoid miss-use
-     */
```

Comment:
Could you handle this one?
Good easy pick

Reply:
Okay! I'm not a developer: coding expertise: -1 🤯